tasks/build-exports-list.js

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import path from 'path';
+import fs from 'fs';
+import fg from 'fast-glob';
+
+const build = (packageJSONPath) => {
+    const exportsPath = path.join(packageJSONPath, '..', 'exports.json');
```

Comment:
I mainly wrote this to help me create them the first time. It's not _really_ to maintain them dynamically. It would be rad if you had thoughts on how to do that, but with there not being a 100% homogenous approach to exports, I wasn't seeing a path.

Reply:
Makes sense, do you mind if I open an issue as a placeholder to think on that in the future?

Reply:
https://github.com/adobe/spectrum-web-components/issues/2463